libnavigation-core/api/current.txt

```diff
@@ -950,7 +952,8 @@ package com.mapbox.navigation.core.routeoptions {
 
 package com.mapbox.navigation.core.routerefresh {
 
-  @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public final class RouteRefreshController {
+  @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public final class RouteRefreshController implements com.mapbox.navigation.core.directions.session.RoutesObserver {
+    method public void onRoutesChanged(com.mapbox.navigation.core.directions.session.RoutesUpdatedResult result);
```

Comment:
It's an interface method, the class overrides it. That's why it's public. But yes, probably we shouldn't make it public. Just create an internal observer so that the user can't invoke `onRoutesUpdated` themselves. Anyway, we can do it in Dash Core only.

Reply:
https://mapbox.atlassian.net/browse/NAVAND-1661